tracer/build/_build/Build.Utilities.cs

```diff
@@ -114,7 +114,7 @@ partial class Build
         .Executes(() => {
 
             var envVars = new Dictionary<string, string> { { "ASPNETCORE_URLS", "http://*:5003" } };
-            envVars.AddTracerEnvironmentVariables(TracerHomeDirectory);
+            envVars.AddTracerEnvironmentVariables(MonitoringHomeDirectory);
```

Comment:
You're really trying to mess up my dev loop here aren't you 😛

Reply:
you know me so well :P